cmd/bootstrap/runner.go

```diff
@@ -214,6 +226,80 @@ func (r *runner) ensurePriorityClass(ctx context.Context, k8sClients k8sclient.I
 	return nil
 }
 
+func (r *runner) installChartMuseum(ctx context.Context, k8sClients k8sclient.Interface) error {
```

Comment:
Why not call it `installAppCatalogs`?

Reply:
That's much nicer. Thx. 

I split out the installing the appcatalogs. Now installChartMuseum just creates the app CR.